src/sys/windows.rs

```diff
@@ -84,6 +84,19 @@ pub(crate) use windows_sys::Win32::Networking::WinSock::{
 pub(crate) const IPPROTO_IP: c_int = windows_sys::Win32::Networking::WinSock::IPPROTO_IP as c_int;
 pub(crate) const SOL_SOCKET: c_int = windows_sys::Win32::Networking::WinSock::SOL_SOCKET as c_int;
 
+macro_rules! export_constants {
+    ($id:ident, $ref:ident) => {
+        pub(crate) const $id: c_int = windows_sys::Win32::Networking::WinSock::$ref as c_int;
+    };
+    ($id:ident) => {
+        export_constants!($id, $id);
+    };
+}
+
+export_constants!(IPV6_MTU_DISCOVER);
+export_constants!(IP_MTU_DISCOVER);
+export_constants!(IP_PMTUDISC_DO);
+export_constants!(IP_PMTUDISC_DONT);
```

Comment:
Can you please remove this macro and use the same style we do for everything else?